src/faq.md

```diff
@@ -202,6 +202,20 @@ Inside or outside of Google, [every Flutter app][FlutterShowcase] uses Dart.
 
 ## Native execution
 
+### Q. Is Dart single-threaded?
+
+No. On native targets, Dart features like isolates and async-await
+can enable multiple threads of execution at any given time.
+The Dart VM will utilize multiple cores of a multi-core processor
+to run those threads concurrently.
+
+[Dart's concurrency architecture](/guides/language/concurrency)
+abstracts the complex, error-prone code of typical shared-memory threading,
+which might explain the misconception that Dart is single-threaded.
+
+This is not quite the case for web apps, though; see
+[Is Dart single-threaded on the web?](#q-is-dart-single-threaded-on-the-web)
```

Comment:
**suggestion:** Simplify the language here.
```suggestion
Dart differs in how it runs on the Web. To learn more, see
[Is Dart single-threaded on the web?](#q-is-dart-single-threaded-on-the-web)
```

Reply:
I feel like "Dart differs in how it runs on the web" is another statement that can be taken out of context like the whole "dart is single-threaded" thing that started this.